src/trio/_tools/gen_exports.py

```diff
@@ -180,22 +180,13 @@ def run_linters(file: File, source: str) -> str:
       SystemExit: If either failed.
     """
 
-    success, response = run_black(file, source)
-    if not success:
-        print(response)
-        sys.exit(1)
-
-    success, response = run_ruff(file, response)
-    if not success:  # pragma: no cover  # Test for run_ruff should catch
-        print(response)
-        sys.exit(1)
-
-    success, response = run_black(file, response)
-    if not success:
-        print(response)
-        sys.exit(1)
+    for fn in (run_black, run_ruff, run_black):
```

Comment:
3144 is merged now, do you still want me to drop the second black run? 

I'm thinking to not change behavior in this PR, and do behavior changes in a follow up

Reply:
yeah, wrote the comment before merging the PR.

This is zero change in behaviour, and "should"ve been in #3144 but since nobody bothered to check out the PR and do the edit there /shrug

Reply:
Ok, I added it